.gitignore

```
@@ -8,6 +8,7 @@
 *.avi
 *.ogv
 *.mesh
+*.inp
```

What kind of file extension is this?

Also, should we add
```suggestion
*.inp
**/p4est_data
```
to the list?

---

This is the Abaqus mesh file extension.
I don't think it's necessary to add `p4est_data` since this file is always in the `out` folder, which is ignored anyway.

---

OK